.github/workflows/not_working/image.yaml

```diff
@@ -0,0 +1,86 @@
+# Current error: https://github.com/cloudfoundry/app-autoscaler-release/actions/runs/3742155200/jobs/6352722260#step:4:106
+#
```

Comment:
Should we split the workflows into `not_working`, `ok` and `original`?

I think I would prefer them to stay at their original location and those that have been ported can be identified by `runs-on: self-hosted`.